DESCRIPTION

```diff
@@ -1,11 +1,16 @@
 Package: MEMC
 Type: Package
-Title: Microbial Explicit Model Comparison: a R package supporting model exploration
+Title: Microbial Explicit Model Comparison
 Version: 0.1.0.999999
-Authors@R: c(person("Kalyn", "Dorheim", ,email = "kalyn.dorheim@pnnl.gov", role = "aut", comment = c(ORCID = "0000-0001-8093-8397")),
-          person("Jianqiu", "Zheng", ,email = "jianqiu.zheng@pnnl.gov", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-1609-9004")))
+Authors@R: c(
+    person("Kalyn", "Dorheim", email = "kalyn.dorheim@pnnl.gov", 
+        role = "aut", comment = c(ORCID = "0000-0001-8093-8397")),
+    person("Jianqiu", "Zheng", email = "jianqiu.zheng@pnnl.gov", 
+        role = c("aut", "cre"), comment = c(ORCID = "0000-0002-1609-9004"))
+    )
```

Comment:
@bpbond add you info?

Reply:
Let's hold off on that for now, and decide later. Thank you!